### DISCUSSION TITLE
Homebrew Analytics Data “OS Versions for Events” and “Cask Install Events” is NaN

### DISCUSSION THREAD

**Discussion by Undefined-User** [Homebrew/discussions#4675] [Everyday usage]
### Output of `brew config`

```shell
NaN
```

### Output of `brew doctor`

```shell
NaN
```

### Description of issue

https://formulae.brew.sh/analytics/cask-install/30d/
https://formulae.brew.sh/analytics/cask-install/90d/
https://formulae.brew.sh/analytics/cask-install/365d/

https://formulae.brew.sh/analytics/os-version/30d/
https://formulae.brew.sh/analytics/os-version/90d/
https://formulae.brew.sh/analytics/os-version/365d/

**Comment by SMillerDev** [comment]
Yeah, it seems InfluxDB changed something in their data output.
